torchtitan/components/dataloader.py

```diff
@@ -17,7 +17,7 @@
 from torchtitan.tools.logging import logger
 
 
-class DataloaderStopIteration(StopIteration):
+class DataloaderStopIteration(Exception):
```

Comment:
Can you add a comment mentioning PEP 479 to avoid people in the future changing this Exception back to StopIteration?